HLTrigger/HLTanalyzers/test/test_hltrigreport_lumi_job.py

```diff
@@ -0,0 +1,4 @@
+from test_hltrigreport_base_cfg import process
+
+process.hlTrigReport.resetBy = "lumi"
+process.hlTrigReport.reportBy = "job"
```

Comment:
does it really make sense to reset every lumi and report only at the end of the job ?

Reply:
It would give you the last lumi of the job.
Again, it was what the original coe supported so I kept the same behavior.